app/controllers/markdown_preview_controller.rb

```diff
@@ -0,0 +1,10 @@
+class MarkdownPreviewController < ApplicationController
+
+  def preview
+    @text = params[:text]
+    respond_to do |format|
+      format.js { render "shared/markdown_preview" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.